data/zilliqa/assets/zil1vdc79fg75sth96xnt6xmq5f55swcd5t6m5zhvn/meta.json

```diff
@@ -1,6 +1,18 @@
 {
   "name": "YODA",
   "symbol": "YODA",
+  "web": "http://jody.tilda.ws",
+  "whitepaper": "http://jody.tilda.ws/whitepaper",
+  "supply": 10000,
+  "decimals": 5,
+  "email": "anzellkennard35@gmail.com",
+  "publicTeam": true,
+  "links": {
+    "twitter": "https://twitter.com/Jody96718451",
+    "telegram": "https://t.me/Yodaco",
+    "linkedin": "https://www.linkedin.com/in/anzell-kennard-87817021b",
+    "facebook": "https://www.facebook.com/anzell.kennard"
```

Comment:
ditto, this is for facebook company profiles

Reply:
same as this - there is plenty of example tokens using a personal page for when im the sole developer?

Reply:
cryptometa/data/zilliqa/assets/zil14d6wwelssqumu6w9c6kaucz2r57z34cxuh96lf/meta.json
this is shards.. using a personal facebook?
https://www.facebook.com/renat.akhmadishin

example.